examples/hot-module-replacement/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "backpack-examples-basic",
```

Comment:
Whoops, just fixed with `backpack-example-hot-module-replacement`.

I can rename the whole thing to HMR, which is soooo much shorter!